src/iop/borders.c

```diff
@@ -178,10 +192,10 @@ int legacy_params(dt_iop_module_t *self,
     } dt_iop_borders_params_v1_t;
 
     const dt_iop_borders_params_v1_t *o = (dt_iop_borders_params_v1_t *)old_params;
-    dt_iop_borders_params_v3_t *n =
```

Comment:
You don't need to modify old legacy versions. The `legacy_params` routines is called many time until the last version of the module is found.